deepreg/model/network/ddf_dvf.py

```diff
@@ -17,7 +17,7 @@ def ddf_dvf_forward(
     moving_label: (tf.Tensor, None),
     moving_image_size: tuple,
     fixed_image_size: tuple,
-    output_dvf: bool,
+    output_model: (str, None),
```

Comment:
warning: potential conflicts with #537, we will sort it out later, no worries

Reply:
ok, I leave it like this